hedera-node/hedera-mono-service/src/main/java/com/hedera/services/context/properties/GlobalDynamicProperties.java

```diff
@@ -362,6 +365,10 @@ public AccountID fundingAccount() {
         return fundingAccount;
     }
 
+    public Address fundingAccountAddress() {
+        return EntityIdUtils.asTypedEvmAddress(fundingAccount);
```

Comment:
_Nit_ - instead of creating a new `Address` for every time this is called, we could set a `Address fundingAccountAddress` field in `reload()`.

Reply:
DONE